plugins/uefi-capsule/efi/generate_binary.py.in

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3
+# pylint: disable=missing-docstring
+#
+# SPDX-License-Identifier: LGPL-2.1+
+
+import subprocess
+from subprocess import PIPE
+import sys
+
+
+def get_positional_args():
+    if len(sys.argv) != 5:
+        print("Not enough arguments")
+        sys.exit(1)
+
+    return sys.argv[1], sys.argv[2], sys.argv[3], sys.argv[4]
+
+
+def run_objcopy():
+    objcopy_cmd = subprocess.run(
+        ["sh", "-c", "command -v @EFI_OBJCOPY@"],
```

Comment:
i do wonder if we could just add this as an argument rather than doing the .py.in thing.

Reply:
Right, will change this as well.